config/app/com/microsoft/azure/iotsolutions/uiconfig/webservice/auth/UserManagementClient.java

```diff
@@ -29,6 +30,10 @@ public UserManagementClient(final IClientAuthConfig config, final WSClient wsCli
     @Override
     public CompletionStage<List<String>> getAllowedActions(String userObjectId, List<String> roles) {
         String url = String.format("%s/users/%s/allowedActions", this.userManagementServiceUrl, userObjectId);
+        if (roles == null) {
```

Comment:
We could also initialize roles to an empty list in UserClaims.java:
```
public class UserClaims {
    private String userObjectId;
    private List<String> Roles;

    public UserClaims() {
    }
```

Reply:
even we initialize list here, it still could be set null throught setRoles() method.